src/reducers/alertsSlice.ts

```diff
@@ -1,55 +1,52 @@
 /*
  * This file is part of ciboard
-
+ *
  * Copyright (c) 2021 Andrei Stepanov <astepano@redhat.com>
- * 
+ * Copyright (c) 2023 Matěj Grabovský <mgrabovs@redhat.com>
+ *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 3 of the License, or (at your option) any later version.
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  * Lesser General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
 import _ from 'lodash';
-import update from 'immutability-helper';
+import { createSlice } from '@reduxjs/toolkit';
 
 import {
-    POP_ALERT,
-    PUSH_ALERT,
     IStateAlerts,
-    ActionsAlertsType,
+    ActionPopAlert,
+    ActionPushAlert,
 } from '../actions/types';
 
 const INITIAL_STATE: IStateAlerts = {
     alerts: [],
 };
 
-export const alertsReducer = (
-    state = INITIAL_STATE,
-    action: ActionsAlertsType,
-): IStateAlerts => {
-    switch (action.type) {
-        case PUSH_ALERT: {
-            const { key, variant, title } = action.payload;
-            return update(state, {
-                alerts: { $push: [{ key, variant, title }] },
-            });
-        }
-        case POP_ALERT: {
+export const alertsSlice = createSlice({
+    name: 'alerts',
+    initialState: INITIAL_STATE,
+    reducers: {
+        popAlert: (state, action: ActionPopAlert) => {
             const { key } = action.payload;
-            return update(state, {
-                alerts: { $set: _.reject(state.alerts, { key }) },
-            });
-        }
-        default:
-            return state;
-    }
-};
+            state.alerts = _.reject(state.alerts, { key });
+        },
+        pushAlert: (state, action: ActionPushAlert) => {
+            const { key, variant, title } = action.payload;
+            state.alerts.push({ key, variant, title });
+        },
+    },
+});
+
+export const { popAlert, pushAlert } = alertsSlice.actions;
+
+export const alertsReducer = alertsSlice.reducer;
```

Comment:
I think this is confusing. Raming file `alertsReducer.ts` to `alertsSlice.ts` 
and export `reducer`

Reply:
It exports not only the reducers but also the action creators, here `popAlert()` and `pushAlert()`.

Reply:
I do not understand what the reason to rename file. Is it related to this major bump release?
If yes, then I need better explanation. Maybe some more extended justification will help.
If it is not related to transition: 7->8 than I ask to do not do this. Since it really complicates PR review process.